lib/outdated.js

```diff
@@ -87,6 +87,9 @@ class Outdated extends ArboristWorkspaceCmd {
     // sorts list alphabetically
     const outdated = this.list.sort((a, b) => a.name.localeCompare(b.name, 'en'))
 
+    if (outdated.length > 0)
```

Comment:
Sorry, I don't think I was clear enough.  There doesn't need to be an if statement at all. The setting of process.exitCode can happen after the next if statement completes. If the length is 0, this function returns.  If it doesn't, then we set the exitCode.

Reply:
The length being zero does not guarantee a return.

If `this.npm.config.get('json') === true`, the function will not return (it's a `&&` condition), so we can't set the `exitCode` to `1` unconditionally actually.

Reply:
🤦 you are totally right.  Your if statement there is the correct one. Thanks for your patience with me here.